src/Mod/TechDraw/App/DrawViewPart.cpp

```diff
@@ -348,6 +348,8 @@ void DrawViewPart::onChanged(const App::Property* prop)
 void DrawViewPart::partExec(TopoDS_Shape shape)
 {
 //    Base::Console().Message("DVP::partExec()\n");
+    if(geometryObject)
```

Comment:
This I have no idea if it is the correct fix, but partExec leaks an object on every run

Reply:
Seems to me it's correct.  Moreover, partExec() is only called from lines 291-308 in the same file, and originally there has been a similar delete-if-not-nullptr functionality.  But, as seen from the code on those lines, after it has been touched, the author has forgotten to add the deletion logic to the other call.  I think it avoids mistakes to have the deletion logic inside the function like you propose.  Note, that lines 301-305 of DrawViewPart.cpp should be removed (they will be superfluous as they have the same functionality as your proposed addition)

Reply:
See 542bf23cb0